src/Veldrid/RasterizerStateDescription.cs

```diff
@@ -36,18 +48,27 @@ public struct RasterizerStateDescription : IEquatable<RasterizerStateDescription
         /// <param name="frontFace">Controls the winding order used to determine the front face of primitives.</param>
         /// <param name="depthClipEnabled">Controls whether depth clipping is enabled.</param>
         /// <param name="scissorTestEnabled">Controls whether the scissor test is enabled.</param>
+        /// <param name="depthBiasEnabled">Controls whether depth bias is enabled.</param>
+        /// <param name="depthBiasConstant">Controls how much constant depth bias to apply.</param>
+        /// <param name="depthBiasSlope">Controls how much slope-scaled depth bias to apply.</param>
         public RasterizerStateDescription(
             FaceCullMode cullMode,
             PolygonFillMode fillMode,
             FrontFace frontFace,
             bool depthClipEnabled,
-            bool scissorTestEnabled)
+            bool scissorTestEnabled,
+            bool depthBiasEnabled = false,
```

Comment:
These parameters need to be added with a new constructor overload (and as non-optional params). The way these are added here is a binary breaking change, because the old overload effectively doesn't exist anymore.